src/App/Resources/AppResources.resx

```diff
@@ -2317,4 +2317,19 @@
   <data name="LogInDenied" xml:space="preserve">
     <value>Log in denied</value>
   </data>
+  <data name="ApproveLoginRequests" xml:space="preserve">
+    <value>Approve login requests</value>
+  </data>
+  <data name="UseThisDeviceToApproveLoginRequests" xml:space="preserve">
+    <value>Use this device to approve login requests made from other devices.</value>
+  </data>
```

Comment:
IMO, the key should be the same as the value unless very lengthy value, so here is not that long so: `UseThisDeviceToApproveLoginRequestsMadeFromOtherDevices `